codalab/lib/bundle_cli.py

```diff
@@ -2091,11 +2091,16 @@ def do_add_command(self, args):
 
         elif args.item_type == 'bundle':
             for bundle_spec in args.item_spec:
-                source_client, source_spec = self.parse_spec(bundle_spec)
+                if '//' in bundle_spec:
```

Comment:
Can we consolidate this logic?  Lack of consolidation was the reason we didn't catch this the first time around...

Reply:
Good point. Will refactor the general client/worksheet resolution and target parsing logic.